kroxylicious-filters/kroxylicious-encryption/src/test/java/io/kroxylicious/filter/encryption/inband/InBandKeyManagerTest.java

```diff
@@ -178,7 +178,7 @@ void encryptionRetry() {
 
         List<TestingRecord> encrypted = new ArrayList<>();
         List<TestingRecord> initial = List.of(record, record2);
-        CompletionStage encrypt = km.encrypt("topic", 1, new EncryptionScheme<>(kekId, EnumSet.of(RecordField.RECORD_VALUE)),
+        CompletionStage encrypt = km.encrypt("topic", 1, new EncryptionScheme(kekId, EnumSet.of(RecordField.RECORD_VALUE)),
```

Comment:
missing generic type

```suggestion
        var encrypt = km.encrypt("topic", 1, new EncryptionScheme(kekId, EnumSet.of(RecordField.RECORD_VALUE)),
```